packages/components/date-picker-overlay/README.md

```diff
@@ -0,0 +1,156 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - date-picker-overlay</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-date-picker-overlay--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.date-picker-overlay">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.date-picker-overlay" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.date-picker-overlay?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.date-picker-overlay" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.date-picker-overlay">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.date-picker-overlay" height="20"/>
+		</a>
+</p>
+
+<style>
+  table {
+        width:100%;
+  }
+</style>
+
+## ➤ Properties
+
+| Property          | Attribute         | Type     | Default | Description                                |
+| ----------------- | ----------------- | -------- | ------- | ------------------------------------------ |
+| dir               | dir               | String   | ltr     | Direction of the component 'rtl' or 'ltr'. |
+| translations      | translations      | Object   |         |                                            |
+| firstDay          | firstDay          | Number   |         |                                            |
+| pageDate          | pageDate          | String   |         |                                            |
+| selectedDate      | selectedDate      | String   |         |                                            |
+| deselectable      | deselectable      | Boolean  | false   |                                            |
+| disabledDateCheck | disabledDateCheck | Function |         |                                            |
+
+## ➤ Events
+
+| Name                 | Description |
+| -------------------- | ----------- |
+| selected-date-change |             |
```

Comment:
Looks like a bug in docs generator, same event name is duplicated.

Reply:
fixed it by just using the emit helper method once